Sources/SwiftParser/Attributes.swift

```diff
@@ -650,7 +650,8 @@ extension Parser {
     let unexpectedBeforeAccessor: RawUnexpectedNodesSyntax?
     let accessor: RawTokenSyntax?
     if period != nil {
-      (unexpectedBeforeAccessor, accessor) = self.expect(.keyword(.get), .keyword(.set), default: .keyword(.get))
+      (unexpectedBeforeAccessor, accessor) = self.expect(.keyword(.get), .keyword(.set), .keyword(._modify),
+                                                         default: .keyword(.get))
```

Comment:
You will probably need to run swift-format on this to format it correctly, otherwise CI will reject the PR.

Reply:
Yeah, had some local issues with building it, now resolved